### PR TITLE
Remove workarounds

### PR DESCRIPTION
**Pull request by wli3**
Please review by commit. These are 3 additional changes related to WindowsDesktop SDK.

8ecabfc is to solve https://github.com/dotnet/sdk/issues/12403 (currently has a workaround in SDK)
7baee74 is solve a simple mistake
f309b4b is moved due to dotnet/sdk@d21aaa5

Tested against _WPF-Samples\Graphics\Point_. And all SDK unit tests

SDK side change https://github.com/dotnet/sdk/pull/12459

### CODE REVIEW

**Comment by wli3** [comment]
@ryalanms could you review this?

also @dsplaisted @sfoslund

File: packaging/Microsoft.NET.Sdk.WindowsDesktop/targets/Microsoft.NET.Sdk.WindowsDesktop.props [hunk 1]
```diff
@@ -121,10 +121,7 @@
     
     Detect these situations and skip updates to @(SupportedTargetFramework) etc. 
   --> 
-  <ItemGroup Condition="('$(UseWPF)' == 'true' Or '$(UseWindowsForms)' == 'true') And 
-                        '$(_TargetFrameworkVersionValue)' != '' And
-                        '$(_TargetFrameworkVersionValue)' != '$(_UndefinedTargetFrameworkVersion)' And 
-                        '$(_TargetFrameworkVersionValue)' >= '$(_WindowsDesktopSdkTargetFrameworkVersionFloor)'">
+  <ItemGroup Condition=" '$(_EnableWindowsDesktopSupportUnsupportedTargetFramework)' == 'true' ">
```
Comment (dsplaisted) [inline packaging/Microsoft.NET.Sdk.WindowsDesktop/targets/Microsoft.NET.Sdk.WindowsDesktop.props:124]: I think renaming this to something like `_RemoveSupportedTargetFrameworksForWindowsDesktop` would be clearer.

Comment (vatsan-madhavan) [reply packaging/Microsoft.NET.Sdk.WindowsDesktop/targets/Microsoft.NET.Sdk.WindowsDesktop.props:124]: Isn't the *intent* here better described as `_RemoveUnsupportedTargetFrameworksForWindowsDesktop` ?

Comment (vatsan-madhavan) [reply packaging/Microsoft.NET.Sdk.WindowsDesktop/targets/Microsoft.NET.Sdk.WindowsDesktop.props:124]: Where is `_EnableWindowsDesktopSupportUnsupportedTargetFramework` being set/calcualted to `true` ?

Comment (wli3) [reply packaging/Microsoft.NET.Sdk.WindowsDesktop/targets/Microsoft.NET.Sdk.WindowsDesktop.props:124]: `_RemoveUnsupportedTargetFrameworksForWindowsDesktop` sounds more accurate

Comment (wli3) [reply packaging/Microsoft.NET.Sdk.WindowsDesktop/targets/Microsoft.NET.Sdk.WindowsDesktop.props:124]: > Where is _EnableWindowsDesktopSupportUnsupportedTargetFramework being set/calcualted to true ?

here https://github.com/dotnet/wpf/blob/7baee74daa23d1e8933a35085821934b0788246d/packaging/Microsoft.NET.Sdk.WindowsDesktop/targets/Microsoft.NET.Sdk.WindowsDesktop.targets#L73

Comment (vatsan-madhavan) [reply packaging/Microsoft.NET.Sdk.WindowsDesktop/targets/Microsoft.NET.Sdk.WindowsDesktop.props:124]: Every time we use this trick, we should document what we are doing 😄.

Comment (wli3) [reply packaging/Microsoft.NET.Sdk.WindowsDesktop/targets/Microsoft.NET.Sdk.WindowsDesktop.props]: renamed to _RemoveUnsupportedTargetFrameworksForWindowsDesktop

**Pull request merged**